experiment/measurer.py

```diff
@@ -392,6 +395,7 @@ def is_cycle_unchanged(self, cycle: int) -> bool:
         def copy_unchanged_cycles_file():
             result = gsutil.cp(exp_path.gcs(self.unchanged_cycles_path),
                                self.unchanged_cycles_path,
+                               parallel=False,
```

Comment:
Can we please just set to parallel to False by default, this set of bugs is getting annoying.

Reply:
Definitely, I just thought it would be better to do in another CL.
WDYT?

Reply:
Just 7 places, lets just do it here.

Reply:
Doing this now but it is more than 7 places if we want to make sure that every place that was previously using parallel=True implicitly still uses it.

Reply:
this is fine, you are breaking measurer anyway soon, so this does not matter much.

Reply:
Done. Does latest iteration look good to you?